ccp/lib/utils/parsing.py

```diff
@@ -0,0 +1,55 @@
+import ast
+import base64
+import json
+
+
+def encode(data):
+    """
+    Applyies base64 encoding to passed string
+    :param data: The string to encode
+    :type data str
+    :return: Encoded string.
```

Comment:
remove the unneeded dot (.) at the end of `Encoded string.`

Reply:
Removed